modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -204,6 +205,11 @@ export interface BitgoHeldBackupKeyShare {
   keyShares: ApiKeyShare[];
 }
 
+export interface BackupKeyShare {
```

Comment:
I kinda like having a differentiation in types for the shares held by client vs bitgo, but yea I see that is not very scalable when we start supporting multiple KRS's. 

Also this might require a lot of work actually. Since the `userHeldKeyshare` is of type `KeyShare` which is accepted as is in many helper functions currently which work directly on the `pShares` and the `nShares`. 

Creating a new common type will require refactoring all those helpers, and i'm not sure if i have enough of the cryptographic context in the sdk to do that refactor.